tests/test_indexing.py

```diff
@@ -27,4 +27,14 @@ def test_warc_ipwbIndexerBrokenWARCRecord():
     assert ipwbTest.countCDXJEntries(cdxj) == 1
 
 
+# WARC/1.1 allows for dates of length that are not easily converted
+# to 14-digits. This test highlights the failures from a WARC
+# exhibiting these dates.
+@pytest.mark.ipwbIndexerVariableSizedDates
+def test_warc_ipwbIndexerVariableSizedDates():
```

Comment:
Why mixed-case function name?

Reply:
Aligning naming to other tests. This PR is not about revising the naming scheme but to add another test.

Reply:
Do you anticipate a separate PR fixing all the symbol names across the board?

Reply:
It is something I would like to do -- align with Pythonic syntactic and semantic conventions after being more informed by @anatoly-scherbakov's contributions. This might be best described as a GitHub issue highlighting where the codebase strays away and making a TODO of fixes for conformance (e.g., to PEPs).

Reply:
When I commented on it, I too was thinking about @anatoly-scherbakov's argument about fixing things as we encounter them rather than waiting for one big overhaul.

Reply:
Either approach is good, so long as we have a concrete target. My style is affected by conventions of a few past languages, but I am more than willing to adhere. It might be good to have some style check in-place to enforce this. Ideally, it would just fix it like gofmt.